factory/__init__.py

```diff
@@ -17,5 +18,5 @@ def make_core():
     """
     slack_token = os.environ['SLACK_API_TOKEN']
     facade = DBFacade(DynamoDB())
-    slack_client = SlackClient(slack_token)
-    return Core(facade, slack_client)
+    bot = Bot(SlackClient(slack_token))
```

Comment:
I wonder if we should rename the `Bot` class - it's really a general interface for the Slack Web API, not anything specifically bot-related.

Reply:
Yeah. It always messes me up a bit. Put that in as an issue for refactoring?

Reply:
Yep, go ahead. 👍